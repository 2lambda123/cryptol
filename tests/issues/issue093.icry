// Don't show examples, as they can differ depending on the solver version.
// See #1280.
:set show-examples=false
:l issue093.cry
:check
:prove
:sat
