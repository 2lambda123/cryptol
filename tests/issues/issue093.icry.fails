Known issue, see ticket #117.
