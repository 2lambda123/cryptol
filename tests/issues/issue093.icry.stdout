Loading module Cryptol
Loading module Cryptol
Loading module Main
property t0 Using exhaustive testing.
testing...passed 1 tests.
Q.E.D.
property t1 Using random testing.
testing...passed 100 tests.
Coverage: 0.00% (100 of 2^^32 values)
property t2 Using random testing.
testing...passed 100 tests.
Coverage: 0.00% (100 of 2^^64 values)
property f0 Using exhaustive testing.
testing...f0 = False
:prove t0
	Q.E.D.
:prove t1
	Q.E.D.
:prove t2
	Q.E.D.
:prove f0
	f0 = False
:sat t0
	t0 = True
:sat t1
	t1 0x00000000 = True
:sat t2
	t2 0xfffffffe 0xffffffff = True
:sat f0
	Unsatisfiable
